### PR TITLE
Changing interlock names for cryopumps and sds chamber

### PR DESCRIPTION
**Pull request by wwright-slac**
This PR changes CRIXS VAC and CRIXS MOTION interface variable names to account for changes to PV naming structure in CRIXS Vacuum PLC

### CODE REVIEW

File: lcls-plc-crixs-motion/_Config/IO/Device 1 (EtherCAT)/Term 5 (EK1200)/Term 167 (EK1122)/CRIX_VAC_INTF (EL6695-0002).xti [hunk 1]
```diff
@@ -184,22 +184,22 @@
...
-				<Entry Name="bVGC10_CLS" Index="#x6000" Sub="#x01">
+				<Entry Name="bCRYO_VGC_01_CLS" Index="#x6000" Sub="#x01">
 					<Type>BOOL</Type>
 				</Entry>
-				<Entry Name="bVGC10_OPN" Index="#x6000" Sub="#x02">
+				<Entry Name="bCRYO_VGC_01_OPN" Index="#x6000" Sub="#x02">
 					<Type>BOOL</Type>
 				</Entry>
-				<Entry Name="bVGC11_CLS" Index="#x6000" Sub="#x03">
+				<Entry Name="bCRYO_VGC_02_OPN" Index="#x6000" Sub="#x03">
```
Comment (wwright-slac) [inline lcls-plc-crixs-motion/_Config/IO/Device 1 (EtherCAT)/Term 5 (EK1200)/Term 167 (EK1122)/CRIX_VAC_INTF (EL6695-0002).xti:193]: Should be bCryo_VGC_02_CLS

Comment (jyotiphy) [reply lcls-plc-crixs-motion/_Config/IO/Device 1 (EtherCAT)/Term 5 (EK1200)/Term 167 (EK1122)/CRIX_VAC_INTF (EL6695-0002).xti]: Did you tested that interlocks are working as expected ?

Comment (wwright-slac) [inline lcls-plc-crixs-motion/_Config/IO/Device 1 (EtherCAT)/Term 5 (EK1200)/Term 167 (EK1122)/CRIX_VAC_INTF (EL6695-0002).xti]: Double check

File: lcls-plc-crixs-motion/_Config/IO/Device 1 (EtherCAT)/Term 5 (EK1200)/Term 167 (EK1122)/CRIX_VAC_INTF (EL6695-0002).xti [hunk 2]
```diff
@@ -184,22 +184,22 @@
...
 					<Type>BOOL</Type>
 				</Entry>
-				<Entry Name="bVGC11_OPN" Index="#x6000" Sub="#x04">
+				<Entry Name="bCRYO_VGC_02_CLS" Index="#x6000" Sub="#x04">
 					<Type>BOOL</Type>
```
Comment (wwright-slac) [inline lcls-plc-crixs-motion/_Config/IO/Device 1 (EtherCAT)/Term 5 (EK1200)/Term 167 (EK1122)/CRIX_VAC_INTF (EL6695-0002).xti:197]: Here as well

**Pull request merged**